ivadomed/metrics.py

```diff
@@ -6,15 +6,19 @@
 
 
 # METRICS
-def get_metric_fns(task):
+def get_metric_fns(task, eval_params=None):
     metric_fns = [dice_score,
                   multi_class_dice_score,
                   precision_score,
                   recall_score,
                   specificity_score,
                   intersection_over_union,
                   accuracy_score]
-    if task == "segmentation":
+    if eval_params:
+        object_detection_metrics = eval_params['object_detection_metrics']
```

Comment:
Should use dict.get pattern to avoid invalid key error as upstream JSON is never fully validated.

Reply:
since the key 'object_detection_metrics' has already defined in 'config_default.json' file, so it wouldn't be invalid key error show up here.